tests/_test_msui/test_mscolab.py

```diff
@@ -148,7 +170,7 @@ def test_logout(self):
     def test_add_user(self, mockmessage):
         self._connect_to_mscolab()
         self._create_user("something", "something@something.org", "something")
-        assert config_loader(dataset="MSCOLAB_mailid") == "something@something.org"
+        assert config_loader(dataset="MSS_auth").get(self.url) == "something@something.org"
```

Comment:
hmm, that will need a documentation.

Reply:
do we have the keyring stuff documented? Will have a look.

This collects all URL/usernames in the MSS_auth config variable and passwords in the keyring.

Reply:
One could also, likely?, store username *and* password in the keyring and forego this entirely? URLs are otherwise stored in the default_whatever lists.

Reply:
the keyring needs some get attribute to find informations, you get usually by get username the password

Reply:
I added documentation. I am not sure how to "migrate" the changes forward. I would defer that change to another PR.